clang/lib/Driver/Driver.cpp

```diff
@@ -1290,6 +1292,109 @@ void Driver::CreateOffloadingDeviceToolChains(Compilation &C,
         Diag(clang::diag::warn_drv_empty_joined_argument)
             << SYCLTargetsValues->getAsString(C.getInputArgs());
     }
+  }
+  // If the user specified --offload-arch, deduce the offloading
+  // target triple(s) from the set of architecture(s).
+  // Create a toolchain for each valid triple.
+  // We do not support SYCL offloading if any of the inputs is a
+  // .cu (for CUDA type) or .hip (for HIP type) file.
+  else if (HasValidSYCLRuntime &&
+           C.getInputArgs().hasArg(options::OPT_offload_arch_EQ) && !IsHIP &&
+           !IsCuda) {
+    // SYCL offloading to AOT Targets with ``--offload-arch``
+    // is currently enabled only with ``--offload-new-driver`` option.
+    // Emit a diagnostic if ``--offload-arch`` is invoked without
+    // ``--offload-new driver`` option.
```

Comment:
Seeing the double back-tick for quoting options (new to me).  As this is just a comment is there somewhere that this type of marking is significant?